### PR TITLE
[HU] Welcome

### PR DESCRIPTION
**Pull request by Shadowthegod**
Corrected the html form to markdown plus fixed the translation into a more convenient one.

### CODE REVIEW

**Comment by Shadowthegod** [comment]
Huh, sorry for the confusion. This was my very first PR so I accidentally removed the branch required for it.

File: wiki/Welcome/hu.md [hunk 1]
```diff
@@ -1,130 +1,152 @@
...
+[Accuracy results image]: /wiki/shared/Accuracy_results.png "Accuracy of player"
+
+<!-- Title -->
+# Üdvözlet
+![osu!][osu-boxart image]
+
+Először is, szeretnénk teljes szívünkből üdvözölni és adni neked egy ölelést amiért csatlakoztál a közösségünkhöz! Engedd meg hogy elmagyarázzunk egy pár dolgot, mielőtt útnak indulsz. 
+
+## Az útmutató
 -----------
```
Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: remove this

File: wiki/Welcome/hu.md [hunk 2]
```diff
@@ -1,130 +1,152 @@
...
 
-Meg is vagyunk! Most már készen állsz az osu!-val való játékra és a kedvenc zenéid megkeresésére. Ha problémába ütközöl, nyugodtan kérj segítséget a fórumon, vagy a játékon belül a chatben az F8 gomb lenyomásával (a magyar közösség csatornája a \#Hungarian).
+![][welcome04 image]
 
-**SOK SZERENCSÉT ÉS JÓ SZÓRAKOZÁST! :3**
+Meg is vagyunk! Most már készen állsz, hogy te legyél a következő osu! bajnok! Ha bármilyen problémába ütközöl, nyugodtan kérj segítséget a fórumon, vagy a játékon belül (lenyomva az F8 gombot a chat panelhez és a \#help-et választva (a magyar közösség csatornája a \#hungarian).
 
-Játékmódok
-==========
+**SOK SZERENCSÉT ÉS JÓ SZÓRAKOZÁST!**
```
Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: I'm reading a rough translation but what is the meaning for this?

File: wiki/Welcome/hu.md [hunk 3]
```diff
@@ -1,130 +1,152 @@
...
+### Egyjátékos (Solo)
+Szóval mi is pontosan a "Solo" mód? Gyakorlatilag ez az a mód, amit egyedül tudsz játszani akár offline, akár online. Itt nincsenek korlátok, bármely (neked meglévő) mapet játszhatod bármely [moddal][Game_Modifiers wikilink] (néha pontmódosítóknak hívják őket). Újraindíthatod a mapet ha úgy érzed túl alacsony a pontszámod vagy a pontosságod, illetve tarthatsz szünetet is, ha úgy érzed, hogy elfáradt a kezed és/vagy ha úgy érzed te magad is elfáradtál. Megnézheted a helyi vagy online pontszámokat illetve bármelyik visszajátszást is megtekintheted, ha rákattintasz egy pontszámra. Saját visszajátszásaidat az F2 lenyomásával kimentheted, hogy bármikor keresés nélkül újra tudd nézni.
 
-Multi
------
+### Többjátékos (Multiplayer)
+_Teljes magyarázat: [Multi](Multi wikilink)_
```
Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: use `_Teljes magyarázat: [Multi][Multi wikilink]_`

File: wiki/Welcome/hu.md [hunk 4]
```diff
@@ -1,130 +1,152 @@
...
-
-Pontosság
----------
+### Szintek
+A játékos szintje csak az összegyűjtött pontszámot mutatja ranked vagy approved map-eken, a beatmap befejezésétől függetlenül. Röviden, ez mutatja mennyire "tapasztalt" valaki. Viszont, más játékokkal szemben (pl. MMO-k), az osu! **NEM** ad senkinek semmilyen előnyt. Egy természetes adottságokkal rendelkező, rendkívül ügyes 10-es szintű játékos akár megverhet egy 100-as szintűt is (ez azt jelenti, hogy a 100-as szintű játékos mindössze többet játszott online). Ezen felül tartsd észben hogy több pontot kapsz ha magasabb kombót érsz el (több note-ot találsz el egymás után, hibázás nélkül).
 
-*Teljes magyarázat: [Accuracy](HU:Accuracy "wikilink")* <img src="Accuracy.png" title="fig:Right|A játékos pontossága|alt=Egy játékos pontosságát mutató képernyőkép, egy beatmap végigjátszása után." alt="Right|A játékos pontossága|alt=Egy játékos pontosságát mutató képernyőkép, egy beatmap végigjátszása után." width="110" />
+### Teljesítménypontok (PP)
+_Teljes magyarázat: [Performance Points](Performance_Points wikilink)_
```
Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: use `_Teljes magyarázat: [Performance Points][Performance_Points wikilink]_`

File: wiki/Welcome/hu.md [hunk 5]
```diff
@@ -1,130 +1,152 @@
...
+A teljesítménypontok tulajdonképpen egy játékos versenyképességét megmérő pontok. Arra irányul, hogy a tényleges képességre fókuszáljon a játszott idő helyett. A pontok kiszámolása erősen függ a map nehézségétől ami egyedi algoritmusokkal van számolva játékmódonként. Ezt a nehézséget, alapul véve minden pontot, osztályozva van és hozzá van rendelve egy pp érték. A teljesítmény rangod főleg a különböző map-eken lévő teljesítményedtől függ. A legkönyebb út, hogy fejlődj az, ha javítod a pontjaidat nehezebb map-eken. Jegyezd meg, hogy a játékmódok külön vannak számolva \[ami azt jelenti, hogy (osu!/osu!taiko/osu!catch/osu!mania)-t játszva az (osu!/osu!taiko/osu!catch/osu!mania) pp-je növekszik\].
 
-Achievement-ek/Jutalmak
------------------------
+Ha kérdésed van a pp-vel kapcsolatban, akkor olvasd el a [wiki oldalát][Performance_Points#FAQ wikilink] először.
 
-*Teljes magyarázat: [Achievements](HU:Achievements "wikilink")*
+### Pontosság (Accuracy)
+_Teljes magyarázat: [Accuracy](Accuracy wikilink)_
```
Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: use `_Teljes magyarázat: [Accuracy][Accuracy wikilink]_`

File: wiki/Welcome/hu.md [hunk 6]
```diff
@@ -1,130 +1,152 @@
...
+![Egy kép ami egy játékos pontosságát mutatja miután befejezett egy map-et.][Accuracy results image]
 
-**Megjegyzés:** Achievement-eket csak akkor kapsz ha online játszol, bejelentkezve.
+Ez a mérés az átlagos pontosság egy játékos által játszott összes rangsorolt pontszám között. Két típusra oszlik: a játékos össz pontossága és a pp pontossága. A játékon belül a saját paneleden a pp pontosságot látod. Egy játékos össz.-pontosságának megtekintéséhez meg kell látogatnod az "Online Results" részt egy beatmap végigjátszása után, vagy a saját felhasználói oldaladat. Az osu!-ban a pontosság nem csak a célzásod pontosságát, hanem az időzítésed pontosságát is jelöli, szóval fontos hogy a note-okat tökéletes ütemre kapd el. Fordíts extra figyelmet az "Approach Circle"-ökre!
 
-Chat
-====
+### Jutalmak (Achievements)
+_Teljes magyarázat: [Achievements](Achievements wikilink)_
```
Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: use `_Teljes magyarázat: [Achievements][Achievements wikilink]_`

File: wiki/Welcome/hu.md [hunk 7]
```diff
@@ -1,130 +1,152 @@
...
 
-A chat konzol alapjainak ismerése fontos, ha új barátokra akarsz szert tenni. Abban az esetben sem árt ha tudod használni, ha valaki üzenetet küld neked (nem túl valószínű, hacsak nem vagy közismert), tippeket adva vagy segítséget kérve. A képernyő jobb alsó sarkában lévő gombbal nyithatod meg a chatet, de gyorsabb, ha megnyomod az F8 gombot. Alapból meg lesz nyitva néhány fül, például "\#osu!", "\#announce", "\#userlog". **A magyar közösség csatornáját "\#Hungarian" név alatt találod.** Ha beszélgetsz valakivel, az illető neve szintén egy külön fülön lesz. A fülek közti navigálás egyszerű, csak kattints az egyikre, írd be az üzeneted és nyomj ENTER-t. Kérjük, hogy ne spam-elj (rövid üzenetek folyamatos küldözgetése a chatbe), mivel ez automatikusan el fog némítani egy pár percre, illetve "kemény munkád gyümölcsét" is el fogja távolítani. A felhasználók listájának megtekintéséhez nyomd meg az F9 gombot (kibővített chat konzol). Itt szűrőket is használhatsz, hogy leszűkítsd a listát.
+**Megjegyzés:** Achievement-eket csak akkor kapsz, ha online játszol, bejelentkezve.
 
-![](friend-list.png "friend-list.png")
+## Chat
+_Teljes magyarázat: [Chat Console](Chat_Console wikilink)_
```
Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: use `_Teljes magyarázat: [Chat Console][Chat_Console wikilink]_`

File: wiki/Welcome/hu.md [hunk 8]
```diff
@@ -1,130 +1,152 @@
...
+Ezek az alapok, ne légy szégyenlős segítsége kérni akár játékon belül, akár a fórumon. A wiki-t is olvasgathatod, ha jobban meg szeretnéd érteni a játékot.
 
-Hasznos linkek
-==============
+## Beatmapping (Edit)
+_Teljes magyarázat: [Beatmapping](Beatmapping wikilink)_
```
Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: use `_Teljes magyarázat: [Beatmapping][Beatmapping wikilink]_`

File: wiki/Welcome/hu.md [hunk 9]
```diff
@@ -1,130 +1,152 @@
...
 
-Itt van egy pár hasznos link az utazásod megkezdéséhez:-
+Az osu! bárki számára lehetővé teszi a beatmap-ek készítését. Ha úgy gondolod kreatív vagy és szívesen tanulsz, akkor neki is láthatsz. A kezdéshez egyszerűen válasz a főmenüben az "Szerkesztés" (Edit) lehetőséget, majd húzd a zene **.mp3** fájlját a szerkesztőbe. Frissül és már láthatod is az üres mapet a te számod címével. A beatmapping azonban egy _nagyon_ komplikált folyamat amihez szenvedély is kell, így ezen az oldalon bővebben nem lesz róla szó. A legjobb mód az, ha te magad is kipróbálod. [Ezen a linken][Beatmappin/modding: Guide Compendium link] hasznos útmutatókat találhatsz a kezdéshez. A Szerkesztés módot egyébként arra is használhatod, hogy közelebbről szemügyre vegyél egy beatmapet, vagy akár csak azt a részét, ahol mindig elbuksz.  Egyébként **Próbáld kerülni a rank-elt beatmapek szerkesztését**, mert vissza fognak állni **nem rank-elt** állapotba (persze csak a te gépeden), és nem fogsz értük pp-t és pontosságot kapni, amíg nem frissíted őket.
```
Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: broken link: change `[Ezen a linken][Beatmappin/modding: Guide Compendium link]` to `[Ezen a linken][Beatmapping/Modding: Guide Compendium link]`

File: wiki/Welcome/hu.md (hunk 2 above)
Comment (Shadowthegod) [inline wiki/Welcome/hu.md]: It basically means Good luck and have fun! Should I remove it?

Comment (MegaApplePi) [inline wiki/Welcome/hu.md]: 🤔 ... I'll leave it up to you if you want to keep it. But if you do keep it, please use sentence casing:
```
**Sok szerencsét és jó szórakozást!**
```

Comment (Shadowthegod) [inline wiki/Welcome/hu.md]: Removed it. It was rather sudden and was a remnant of the previous translation anyway.

**Comment by TPGPL** [comment]
Sorry but Welcome page is going to be rewritten soon

**Comment by Shadowthegod** [comment]
I had to delete my previous fork so this was dead anyway.

**Pull request closed without merging**